salt/utils/pkg/deb.py

```diff
@@ -1,28 +1,20 @@
-# -*- coding: utf-8 -*-
 """
 Common functions for working with deb packages
 """
 
-# Import python libs
-from __future__ import absolute_import, print_function, unicode_literals
-
-# Import 3rd-party libs
-from salt.ext import six
-from salt.ext.six.moves import range  # pylint: disable=redefined-builtin
-
 
 def combine_comments(comments):
     """
     Given a list of comments, or a comment submitted as a string, return a
     single line of text containing all of the comments.
     """
     if isinstance(comments, list):
-        for idx in range(len(comments)):
-            if not isinstance(comments[idx], six.string_types):
-                comments[idx] = six.text_type(comments[idx])
+        for idx, val in enumerate(comments):
+            if not isinstance(val, str):
+                comments[idx] = str(val)
```

Comment:
**suggestion** also not necessarily part of this PR, but I just ran a bunch of timeit, and

```
comments = [c if isinstance(c, str) else str(c) for c in comments]
```

tends to be faster
```
import timeit

one = """
for idx, val in enumerate(comments):
    if not isinstance(val, str):
        comments[idx] = str(val)
"""

two = "comments = [c if isinstance(c, str) else str(c) for c in comments]"

comments = ['hello']*100

# Empty comments
print(timeit.timeit(one, setup="comments = []"))
print(timeit.timeit(two, setup="comments = []"))

# 100 comments that are nothing but text
print(timeit.timeit(one, setup=f"comments = {comments!r}"))
print(timeit.timeit(two, setup=f"comments = {comments!r}"))

comments = [3, 'hi', (1,2,'hi'), {}]*100
# 100 comments that are int, text, tuple, and dict
print(timeit.timeit(one, setup=f"comments = {comments!r}"))
print(timeit.timeit(two, setup=f"comments = {comments!r}"))

# My output:
0.10693163899850333
0.11029277600027854
5.741915374994278
4.387402191998262
24.307309091003845
16.571208328001376
```